src/input-helper.ts

```diff
@@ -82,6 +82,17 @@ export async function getInputs(): Promise<IGitSourceSettings> {
   result.clean = (core.getInput('clean') || 'true').toUpperCase() === 'TRUE'
   core.debug(`clean = ${result.clean}`)
 
+  // Sparse checkout
+  const sparseCheckout = core.getMultilineInput('sparse-checkout')
+  if (sparseCheckout.length) {
+    result.sparseCheckout = sparseCheckout
+    core.debug(`sparse checkout = ${result.sparseCheckout}`)
+  }
+
+  result.sparseCheckoutConeMode =
+    (core.getInput('sparse-checkout-cone-mode') || 'true').toUpperCase() !==
+    'FALSE'
```

Comment:
```suggestion
  result.sparseCheckoutConeMode =
    (core.getInput('sparse-checkout-cone-mode') || 'true').toUpperCase() === 'TRUE'
```

I think we normally compare with `TRUE` in the `input-helper.ts`

Reply:
I fixed it in my most recent [force-push](https://github.com/actions/checkout/compare/cb762b36652fc2af953619bfd25854b94dd62fce..c4602b42ee2a5f835f91eb29a5f4e55c060afa82#diff-2aa7ebec21c6f88acd853266c94639d7a4724a603fdbb435b142603dcc12808bL93-R94).